src/giftCards/GiftCardSettings/GiftCardSettingsPage.tsx

```diff
@@ -0,0 +1,101 @@
+import { Typography } from "@material-ui/core";
+import Container from "@saleor/components/Container";
+import Form from "@saleor/components/Form";
+import { Grid } from "@saleor/components/Grid";
+import PageHeader from "@saleor/components/PageHeader";
+import Savebar from "@saleor/components/Savebar";
+import useNavigator from "@saleor/hooks/useNavigator";
+import { sectionNames } from "@saleor/intl";
+import { Backlink } from "@saleor/macaw-ui";
+import {
+  GiftCardSettingsExpiryTypeEnum,
+  TimePeriodTypeEnum
+} from "@saleor/types/globalTypes";
+import { getFormErrors } from "@saleor/utils/errors";
+import React from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+
+import { giftCardsListPath } from "../urls";
+import GiftCardExpirySettingsCard from "./GiftCardExpirySettingsCard";
+import { giftCardExpirySettingsCard as expirySettingsMessages } from "./GiftCardExpirySettingsCard/messages";
+import { giftCardSettingsPageMessages as messages } from "./messages";
+import { useGiftCardSettingsUpdateMutation } from "./mutations";
+import { useGiftCardSettingsQuery } from "./queries";
+import { GiftCardSettingsFormData } from "./types";
+import { getGiftCardSettingsInputData } from "./utils";
+
+const GiftCardSettingsPage: React.FC = () => {
+  const intl = useIntl();
+
+  const navigate = useNavigator();
+
+  const { data, loading } = useGiftCardSettingsQuery({});
```

Comment:
We should allow for empty function call if no variables/opts are required, possibly by making the default value `{}`, food for thought.